articles/user-profile/normalized/oidc/index.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Claims for a user profile returned via an OIDC-Compliant authorization flow
+---
+
+# User Profiles Returned from OIDC-Compliant Pipelines
```

Comment:
How can I get this user profile? We need to mention this. Maybe add a  `How to retrieve the User Profile` paragraph, like the other doc has? See here https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims#standard-claims